### PR TITLE
added TEP 002: dumping full model information to hdf5

### PR DESCRIPTION
**Pull request by unoebauer**
This TEP is about the capability to store a self-contained detailed set of model information (including, for example, unfiltered packet information and line details) to an HDF5 file.

### CODE REVIEW

File: TEP002_model_hdf5.rst [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+Responsible
+===========
+
+TBD
+
+Branches and Pull requests
+==========================
+
+N/A
```
Comment (wkerzendorf) [inline TEP002_model_hdf5.rst]: A first implementation for this is in https://github.com/tardis-sn/tardis/pull/435. @unoebauer I can just make a PR to this PR and fill out some of the stuff.

File: TEP002_model_hdf5.rst [hunk 2]
```diff
@@ -0,0 +1,62 @@
+TEP002: Meaningful title
```
Comment (wkerzendorf) [inline TEP002_model_hdf5.rst]: `TEP0002: TARDIS model to HDF5` -- maybe

File: TEP002_model_hdf5.rst [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+information in an HDF5 file. This capability may be an additional feature,
+living next to the possibility to store a minimal set of "essential"
+information about the run (similar to what the to_hdf5 currently does). As an
+alternative the to_hdf5 method may be rewritten to operate in two modes,
+minimal and detailed.
+
+Implementation
+==============
+
+N/A
```
Comment (wkerzendorf) [inline TEP002_model_hdf5.rst]: Suggestion for text:
`For the plasma there is a relatively straightforward way of accomplishing this: Give each property a method to write its current value to an HDF5 file. Some issues need to be looked at like storing single float values, but there are some proposed solutions already for this in the current PR`

**Comment by wkerzendorf** [comment]
@unoebauer - I had some comments for this and am also thinking of offering this as a GSoC project. Do you mind if I do a PR to your PR with the suggested changes?

**Comment by unoebauer** [comment]
@wkerzendorf - sure, knock yourself out :)

File: TEP002_model_hdf5.rst (hunk 2 above)
Comment (unoebauer) [inline TEP002_model_hdf5.rst]: TEP0002: TARDIS hdf5 data storage capabilities

something like this, since it is not exclusive to the model any more?

Comment (wkerzendorf) [inline TEP002_model_hdf5.rst]: sounds great!

**Pull request merged**